lib/src/controls/form/color_picker/color_picker.dart

```diff
@@ -0,0 +1,1183 @@
+import 'dart:math' as math;
+
+import 'package:fluent_ui/fluent_ui.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/rendering.dart';
+
+import 'color_spectrum.dart';
+import 'color_state.dart';
+
+/// Defines the shape of the color spectrum in the [ColorPicker].
+enum ColorSpectrumShape {
+  /// A ring-shaped color spectrum.
+  ring,
+
+  /// A box-shaped color spectrum.
+  box,
+}
+
+/// Defines the color mode used in the [ColorPicker].
+enum ColorMode {
+  /// RGB (Red, Green, Blue) color mode.
+  rgb,
+
+  /// HSV (Hue, Saturation, Value) color mode.
+  hsv,
+}
```

Comment:
I think this can be private as well. What do you think?

Reply:
I agree! There's no need to expose the `ColorMode` to the public API.